src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -46,7 +47,8 @@
     },
     {
       "auth": "Windows",
-      "authenticationType": "WindowsAuth"
+      "authenticationType": "WindowsAuth",
+      "allowUnsecure": true
```

Comment:
Shouldn't the name be Insecure? I'm not sure about disabling it for windows auth

Reply:
@phenning suggested the name and to disable it for windows auth.

Reply:
Actually let’s make this allowUnsecured. I haven’t checked in the implementation of this yet, so we can still make this change. 

As to whether you want to allow windows auth over http, that is actually the current behavior for 2.0 templates, so I think you’d might want to make it an option in 2.1

Reply:
Okay, removing the allowUnsecure option from WindowsAuth